OkienkoStartowe.java

```diff
@@ -0,0 +1,280 @@
+        wJednaStrone.setLayoutX(150);
+        wJednaStrone.setLayoutY(80);
+
+
+        ToggleGroup togglegroup = new ToggleGroup();
+        wObieStrony.setToggleGroup(togglegroup);
+        wJednaStrone.setToggleGroup(togglegroup);
+
+
+        Label biletNie = new Label("Bilety Nie dostepne, podaj inne daty");
+        biletNie.setLayoutX(60);
+        biletNie.setLayoutY(450);
+        biletNie.setTextFill(Color.RED);
+        biletNie.setVisible(false);
+
+        TextField dataWylotu = new TextField();
+        dataWylotu.setPromptText("DATA WYLOTU [Rok-Msc-Dzien]");
+        dataWylotu.setPrefColumnCount(17);
+        dataWylotu.setLayoutX(40);
+        dataWylotu.setLayoutY(270);
+        dataWylotu.setVisible(true);
+        dataWylotu.setOnAction(new EventHandler<ActionEvent>() {
+            @Override
+            public void handle(ActionEvent event) {
+
+                System.out.println("Wylot w dniu: "+ dataWylotu.getText());
+
+               if (los >= 8 & wJednaStrone.isSelected() == true)
+               {
+                   biletNie.setVisible(true);
+
+               }
+
+               if (los<=7 & wJednaStrone.isSelected() == true)
+               {
+                   biletTak.setVisible(true);
+
+               }
+            }
+        });
+
+        TextField dataPrzylotu = new TextField();
+        dataPrzylotu.setPromptText("DATA PRZYLOTU [Rok-Msc-Dzien]");
+        dataPrzylotu.setPrefColumnCount(17);
+        dataPrzylotu.setLayoutX(40);
+        dataPrzylotu.setLayoutY(320);
+        dataPrzylotu.setVisible(true);
+        dataPrzylotu.setOnAction(new EventHandler<ActionEvent>() {
+            @Override
+            public void handle(ActionEvent event) {
+
+                System.out.println("Przylot w dniu: "+dataPrzylotu.getText());
+
+                if (los>=8 & dataWylotu.isVisible())
+                {
+                    biletNie.setVisible(true);
+                    dataPrzylotu.clear();
+                    dataWylotu.clear();
+
+                }
+                else
+                {
+                    biletNie.setVisible(false);
+                    biletTak.setVisible(true);
+
+                }
+            }
+        });
+
+        wJednaStrone.setOnAction(new EventHandler<ActionEvent>() {
+            @Override
+            public void handle(ActionEvent event) {
+                dataPrzylotu.setDisable(true);
+                System.out.println("W jedna strone");
+            }
+        });
+
+        wObieStrony.setOnAction(new EventHandler<ActionEvent>() {
+            @Override
+            public void handle(ActionEvent event) {
+                dataPrzylotu.setDisable(false);
+                dataWylotu.setDisable(false);
+                System.out.println("W obie strony");
+
+            }
+        });
+
+
+        String [] lista = {"Krakow", "Warszawa", "Sztokholm", "Wieden", "Praga", "Amsterdam", "Glasgow", "Istanbul"};

```

Comment:
Tablice są malo elastyczne i niebezpieczne, gdyz mozemy niechcacy nadpisac ktoras z tych wartosci. Proponuje zrobic niemodyfikowalna liste: `List<String> listOfCities = List.of("Krakow", "Warszawa", "Sztokholm", "Wieden", "Praga", "Amsterdam", "Glasgow", "Istanbul");`